src/main/java/views/add_edit_epense_views/AddExpenseV.java

```diff
@@ -24,14 +23,15 @@ public class AddExpenseV extends Component implements ActionListener, LoadMonthM
     private final JComboBox<String> categoryCombo;
```

Comment:
You would need to extend JFrame instead of Component for your JOptionPane.showMessageDialog() to work. This also applies to EditExpenseV, AddCategoryV, EditCategoryV. 

Since we have all classes merged now, you should run Main.java to test your view.